docs/ml-features.md

```diff
@@ -461,6 +461,87 @@ for binarized_feature, in binarizedFeatures.collect():
 </div>
 </div>
 
+## PCA
+
+[PCA](http://en.wikipedia.org/wiki/Principal_component_analysis) is a statistical procedure that uses an orthogonal transformation to convert a set of observations of possibly correlated variables into a set of values of linearly uncorrelated variables called principal components. A [PCA](api/scala/index.html#org.apache.spark.ml.feature.PCA) class trains a model to project vectors to a low-dimensional space using PCA. The example below shows how to project 5-dimensional feature vectors into 3-dimensional principal components.
```

Comment:
Could you please include the link to the Scala API doc within the Scala codetab here:
```
<div data-lang="scala" markdown="1">
See the [Scala API documentation](api/scala/index.html#org.apache.spark.ml.feature.PCA) for API details.
{% highlight scala %}
```

And please do the same for the Java and Python API docs too.